core/libp2p/multi/multistream.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/multi/multistream.hpp"
+
+#include <iterator>
+#include <string_view>
+
+using kagome::common::Buffer;
+using kagome::expected::Error;
+using kagome::expected::Result;
+using kagome::expected::Value;
+
+namespace libp2p::multi {
+
+  Multistream::Multistream(std::filesystem::path codecPath, Buffer data)
+      : multistream_buffer_{} {
+    auto path_str = codecPath.string();
+    multistream_buffer_
+        // varint length, which includes path + '\n' + data size
+        .put(toVarInt(path_str.length() + 1 + data.size()))
+        .put_uint8('/')
+        .put(path_str)
+        .put_uint8('\n')
```

Comment:
I have a feeling that this code has old base commit, because `put_uint8` was renamed to `putUint8` long time ago. You need to rebase onto master

Reply:
Yes, actually multistream got here accidentally, I will delete it from this branch. It has its own, I just didn't open a PR for it yet.